CHANGELOG.md

```diff
@@ -77,7 +77,7 @@
 
 - v1.0.3: Improved version tracking support
 
-- v1.0.2: Added SetUser method for unique user tracking.
+- v1.0.2: Added SetUser method for unique user tracking/customers.
```

Comment:
I wonder if we should avoid retrospectively changing the change log history.

Reply:
Is it better to bump the provider version stating that we've made documentation changes instead? 🤔

Reply:
Probably not necessary because all you've been doing here is changing terminology in documentation and it's done in a really clear way, I think. In the past I haven't bumped the version for doc improvements.